views/new.erb

```diff
@@ -1,4 +1,6 @@
 <form action="/peeps" method="post">
-  <input type="text" name="name" />
+  
+Peep <input type="text" name="name" />
+  Current Time <input type="time" name="created_at" />
```

Comment:
Great that you've added a timestamp for posts. I think maybe you now could update your post '/peeps' route in app.rb so that it retrieves the params[:created_at] as well as the params[:name]